Default.preset

```diff
@@ -127,6 +127,7 @@ SetVisualFXPerformance          # SetVisualFXAppearance
 # DisableChangingSoundScheme    # EnableChangingSoundScheme
 # EnableVerboseStatus           # DisableVerboseStatus
 DisableF1HelpKey                # EnableF1HelpKey
+ShowBuildNumberOnDesktop        # HideBuildNumberOnDesktop

```

Comment:
```suggestion
# ShowBuildNumberOnDesktop      # HideBuildNumberOnDesktop
```
Comment out this tweak in the default preset because most users will most likely not want to have this enabled.